### DISCUSSION TITLE
Add multiple screens with page up down

### DISCUSSION THREAD

**Discussion by hranmuthu** [geovens/gInk#209] [Ideas]
Initially start with page 0. You can pan and draw anything on a page, when page down is pressed, save the current drawing (in memory or on a file) and clear current drawing and create page 1. You can draw again. if you press page down again, save the current page too and create page  2 and so on. You can use page up or down to browse through the pages or create a new page with page down when you reach the last page.  

This will give us very powerful capability to maintain multiple drawings in one session.

**Comment by pubpub-zz** [comment]
@hranmuthu 
I've added this feature in my fork. A beta version is available in this thread https://github.com/pubpub-zz/ppInk/issues/258
